torchvision/transforms/functional_tensor.py

```diff
@@ -828,19 +832,20 @@ def _perspective_grid(coeffs: List[float], ow: int, oh: int):
     theta1 = torch.tensor([[
         [coeffs[0], coeffs[1], coeffs[2]],
         [coeffs[3], coeffs[4], coeffs[5]]
-    ]])
+    ]], dtype=torch.float, device=device)
```

Comment:
For the future: maybe we might want to pass a `dtype` to this function as well, otherwise other functions might fail for `float64`